src/main/java/core/types/HeartRate.java

```diff
@@ -0,0 +1,18 @@
+package core.types;
+
+/**
+ * Created by mikeecb on 17/10/2015.
+ */
+public class HeartRate {
```

Comment:
Do we really need/want a seperate data type for heart rate? And if so wouldn't it perhaps be better to make it an interface?